### ISSUE TITLE
Manual leveling offset wrong on frist move.

### ISSUE THREAD

**Issue by macasero** [Geeetech3D/Smartto-IAR#6]
I you go to Leeling, manual.
Set a offset (because yo know it, for example Z offset = 3m) and press OK to store it.

Then press any leveling position, for example, Position 2.

The nozzle raises, moves to postion, and lowers, but it will be up by the bed for a gap. It doesn´t go back to Z=3mm, It goes to Z = 3+3.

Seams it moved in relative position during offset setting.

You exit manual leveling, enter again, and directly move to Position 2, it really goes this time to the offset Z postion stored.

**Comment by rokrodic** [comment]
Yes there is a very odd manual leveling issue. It behaves very oddly. I have latest A30_V1.37.58 on my A30. You should correct it! It destroyed my superplate! When you move to a new position the Z offset is sometimes set to 0, sometimes it adds up. The error with Z height is everytime different. Even when the Z offset is set and you save it with "OK", it seems to change?!?

Another problem is if I use induction probe. The problem is on the LCD. It displays "-" Z-offset, which is OK, but when you press "noozle" icon it becomes positive. Afterwards it disappears - both numbers on the screen are 0. One should know what was the last Z-offset, so it should be on the screen.

**Comment by MoeweX** [comment]
I validated that the problem described by macasero exists also on my A30 for firmware version 1.37.57

**Comment by SonOfSardaar** [comment]
I can confirm this too. i agree that nozzle should lift up by offset + few more millimeters to avoid scraping against heat bed. i already scratched mine because of this

**Comment by SonOfSardaar** [comment]
Also want to add that when i start manual bed leveling from middle point (5) and then i click point 1 the nozzle does not go down by same offset (z offset) it goes down a bit on point 1 and others but its too far away from the bed.
